### DISCUSSION TITLE
Ödev 2 kodları

### DISCUSSION THREAD

**Discussion by AybukeNr** [gusanmaz/BMB311_OS_2023#4] [Q&A] [answered]
Hocam, thread_create.c, thread_create_simple_args.c, thread_create_with_return_args.c kodları hangi sayfalarda bulunuyor ?

**Comment by gusanmaz** [comment, marked as answer]
Bu kodlari referans alabilirsiniz: https://github.com/remzi-arpacidusseau/ostep-code/tree/master/threads-api

https://pages.cs.wisc.edu/~remzi/OSTEP/threads-api.pdf bu bolumde bu kodlar ele aliniyor aslinda, kitaptaki kodlar ile repodaki kodlar arasinda ufak farkliliklar olabilir.
